Neos.Flow/Classes/Configuration/RouteConfigurationProcessor.php

```diff
@@ -197,6 +197,13 @@ protected function buildSubRouteConfigurations(array $routesConfiguration, array
                         $mergedSubRouteConfiguration['defaults'][$key] = $this->replacePlaceholders($defaultValue, $variables);
                     }
                 }
+                if (isset($mergedSubRouteConfiguration['routeParts'])) {
+                    foreach ($mergedSubRouteConfiguration['routeParts'] as $routePartKey => $routePartValue) {
+                        foreach ($mergedSubRouteConfiguration['routeParts'][$routePartKey]['options'] ?? [] as $optionKey => $optionValue) {
```

Comment:
I think I see now why you used the `?? []` in the first place. You already (subconciously?) were aware this would lead to an ugly "if { foreach { if { foreach {" cascade

Reply:
very conciously so :)